README.md

```diff
@@ -360,38 +357,32 @@ You will be writing your tests using jasmine-node,
 so you will need to write tests against the [v1.3 api](http://jasmine.github.io/1.3/introduction.html).
 
 You should set up $SISYPHUS/spec to mirror your $SISYPHUS/src directory.
-Tests for, e.g., $SISYPHUS/src/design_docs/boulder/lib/actions.coffee 
-should go in $SISYPHUS/src/design_docs/boulder/lib/actionsSpec.coffee.
-The `Spec` postfix is needed so jasmine-node
+Tests for, e.g., $SISYPHUS/src/design_docs/boulder/lib/app.coffee 
+should go in $SISYPHUS/src/design_docs/boulder/lib/appSpec.coffee.
+The `Spec` suffix is needed so jasmine-node can find your 
 
 You should make liberal use of jasmine spys to mock and spy on external dependencies.
 
 There is already a .travis.yml file in your project skeleton.
-All you need to do is turn your repo on in travis-ci.
+All you need to do is enable your repo on both travis-ci.org and coveralls.io.
 
 ### 5. Implement CouchDB actions
-We will define our actions in `src/design_docs/boulder/lib/actions.coffee`.
-We need to define:
-  1. how to actually do the action, and
-  2. how to validate when an action is allowed
+
+We need to:
```

Comment:
These steps here are no longer in couch, so the head `### 5. Implement CouchDB actions` is inaccurate, right?